src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        for (User checkAllUsersCredit: getUsers()) {
+            if (checkAllUsersCredit.getEmail().equals(email)
+                    && checkAllUsersCredit.getPassword()
+                    .equals(password)) {
+                return true;
+            }
+        }
```

Comment:
it's the same logic as in `findByEmail()` but with password check
think how you can use logic from `findByEmail()` here